testing/end_to_end_tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
```

Comment:
consider changing file name to fit conventions (smth like `test_config`), but nbd